packages/devextreme/scss/widgets/fluent/progressBar/_index.scss

```diff
@@ -60,7 +60,7 @@
   }
 
   .dx-progressbar-status {
-    color: $base-foreground-disabled;
+    color: $base-text-color;
```

Comment:
Progressbar status should have $base-foreground-disabled color. It's correct. Please, revert this change.

Reply:
fixed